### DISCUSSION TITLE
Add inline component blocks

### DISCUSSION THREAD

**Discussion by willemmulder** [keystonejs/keystone#7499] [Feature requests]
As per the discussion with @raveling in Slack.

This is a feature request for inline component blocks

> Is it possible to have inline custom component blocks in the document field? I want to replicate the Link functionality in the editor. I.e. a user can select a piece of text and then click 'Link' to apply the inline custom component block to it.
> The custom component would then wrap that text and displays a URL field on click, just like the normal Link functionality.

With @raveling responding

> they’re definitely something we want to do. We need to update our public roadmap :sweat_smile:

Thanks!

**Comment by stuible** [comment]
We desperately need this for our web app.  Would love to see this come to Keystone CMS!
